postgres_backup/README.md

```diff
@@ -0,0 +1,40 @@
+# postgres\_backup
+
+Creates a cron job that runs a docker container to backup a postgres database onto nfs.
+
+## Requirements
+
+Docker must be running on the host where this role is deployed.
+
+## Usage
+
+It requires the following variables to setup the backup cron job:
+
+- `database_image`: name of the postgres image to use for performing the backup eg. "postgres:9.5"
+- `database_host`: host the database is running on
+- `database_name`: name of the database to backup
+- `database_username`: username to use to log into the database
+- `database_password`: password to use to log into the database
+- `nfs_mount_device`: device where we will backup the database to
+- `nfs_mount_dir`: name of the directory where the nfs device will be mounted
+
+Optional variables customizing backup operation:
+
+- `database_port`: Port postgres is listening on. Defaults to standard postgres port 5432
+- `backup_filename_prefix`: Prefix to put in front of the backup filenames.
+
+
+## Example:
+
+```
+    - role: postgres_backup
```

Comment:
It looks like there are several pieces that assume `become: true`, so please note if that's a requirement (or annotate it on the steps where it is required)

We should be more deliberate about superuser privileges. Obviously installing the NFS mount will require that privilege, but the cron task does not. Perhaps one of the steps here could be to add a local user (in the docker group) that owns the cron job and runs it?

Reply:
Changed role to add a user that performs the backup. Added a note to the README about `become` requirement.